SCHEDULE.md

```diff
@@ -1,29 +0,0 @@
-# Frontend Course 2019
```

Comment:
Maybe no need to delete this file, just remove all content except an empty schedule table template?
After the first lecture, we will create this file again)

Reply:
Good point! added the file back.